src/chigame/api/tests/factories.py

```diff
@@ -29,14 +31,19 @@ class Meta:
     year_published = Faker("pyint", min_value=1900, max_value=2023)
 
     rules = Faker("text", max_nb_chars=1000)
+
     min_players = Faker("pyint", min_value=1, max_value=10)
-    max_players = Faker("pyint", min_value=1, max_value=10)
+
+    # LazyAttribute allows setting a field's value based on other fields at runtime.
+    # In this case, we want max_players to be at least min_players, but no more than 10.
```

Comment:
Great clear comments. Immediately makes the idea of a `LazyAttribute` clear and easy to understand.